native/src/components/FeedbackButtons.tsx

```diff
@@ -0,0 +1,77 @@
+import SadIcon from 'integreat-app/assets/icons/negative-feedback.svg'
+import HappyIcon from 'integreat-app/assets/icons/positive-feedback.svg'
+import React, { ReactElement } from 'react'
+import { useTranslation } from 'react-i18next'
+import { TouchableOpacity } from 'react-native'
+import styled from 'styled-components/native'
+
+import Text from './base/Text'
+
+const Container = styled.View`
+  flex-direction: row;
+  justify-content: center;
+  align-items: center;
+  margin: 12px 0;
+`
+
+const FeedbackTouchableOpacity = styled(TouchableOpacity)`
+  align-items: center;
+  border-radius: 18px;
+  width: 100px;
+  height: 80px;
+  justify-content: center;
+`
+const CardShadow = styled.View<{ $active: boolean | null }>`
+  background-color: ${props => (props.$active ? `${props.theme.colors.themeColor}` : 'white')};
+  border-radius: 18px;
+  shadow-color: #171717;
```

Comment:
Do we actually need a new color for this?

Reply:
use background color